cluster/os-3.9.0/README.md

```diff
@@ -28,10 +24,32 @@ node01    Ready     master    1h        v1.9.1+a0ce1bc657
 node02    Ready     <none>    46s       v1.9.1+a0ce1bc657
 ```
 
+## OpenShift Web Console
+
+If you want to get access to OpenShift web console you will need to add line to `/etc/hosts`
```

Comment:
are you sure the etc/hosts was just required for web console support?

Reply:
Yes, definitely. I never had that entry and only had to add it for the webconsole. Also our CI servers don't have it.